test/epochtalk_server_web/controllers/thread_test.exs

```diff
@@ -21,14 +30,25 @@ defmodule Test.EpochtalkServerWeb.Controllers.Thread do
     admin_threads = build_list(3, :thread, board: admin_board, user: admin_user)
     super_admin_threads = build_list(3, :thread, board: super_admin_board, user: super_admin_user)
 
+    thread = build(:thread, board: board, user: user)
+    admin_thread = build(:thread, board: admin_board, user: admin_user)
+    admin_created_thread = build(:thread, board: board, user: admin_user)
+    super_admin_thread = build(:thread, board: super_admin_board, user: super_admin_user)
+
     {
       :ok,
+      user: user,
```

Comment:
don't need to return user here, not modified

Reply:
This is used on line 557 I believe

"given thread that authenticated user moderates, does purge thread"